### PR TITLE
learning-hour: Domain Oriented Observability 

### PR DESCRIPTION
**Pull request by PierrickBlons**
Hello,

This is a draft on a learning hour I've experienced this week. The goal is to introduce the concept of domain oriented observability described by Pete Hodgon on martin's Fowler blog - https://martinfowler.com/articles/domain-oriented-observability.html

It would be great to have feedback on the current description of the learning hour please :)

The associated Kata is still in progress, I gathered feedbacks when playing the learning hour this week within my team that should be addressed very soon (as long as the kata description coming very soon) - https://github.com/PierrickBlons/reservation-kata

### CODE REVIEW

**Comment by PierrickBlons** [comment]
Everything things ready now for final review :)
The underlying kata is awaiting a final MR to include the approval tests in csharp as there are still discussions with @LarsEckart who is translating it in Java but should be merged very soon.
Let me know if there is anything to improve in this learning hour.

**Comment by emilybache** [comment]
Apologies for the delay, I'm glad you mailed me. I hadn't noticed you'd updated this PR. It's looking very good I just have a couple of comments I will put inline

File: _learning_hours/legacy/domain_observability.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+# Domain Oriented Obervability
+
+It's useful to have logging and metrics in our code to understand how it's behaving in production. But sometimes those little calls to loggers or metrics class start to spread all across the code base. It's quite common to see methods with more code dedicated to observability than code actually solving the problem.
+
+In this learning hour we will see how to improve the understandability of code that requires observability.
+
+### Learning Goals
+
+- Improving the readability of the code
```
Comment (emilybache) [inline _learning_hours/legacy/domain_observability.md:18]: Learning goals should begin with a verb from the "Bloom taxonomy" - I would suggest  "recognize when a domain probe would increase code readability" and "use the extract method refactoring"

File: _learning_hours/legacy/domain_observability.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+* 15 min concept: Introduce domain oriented observability refactoring
+* 30 min concrete: Do Reservation Kata in pairs
+* 10 min conclusions: Discussion about this technique in your current code base
+
+### Connect: What is observability?
+Based on the [Pick only the correct items](https://sammancoaching.org/activities/connect/pick_the_correct_items_on_the_list.html) activity 
+
+Share to the group the some propositions to answer the question "What is observability?".
+
+Example:
```
Comment (emilybache) [inline _learning_hours/legacy/domain_observability.md:33]: Could you give the facilitator a clue what the expected correct answers might be?

**Comment by PierrickBlons** [comment]
Thanks @emilybache for the feedbacks 😃 Everything should be fine now.

**Comment by emilybache** [comment]
Thankyou!

**Pull request merged**